cves/CVE-2003-0083.yml

```diff
@@ -47,21 +50,19 @@ fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
-   - commit: 8bbcf862a6bb9394dfcf505f737b7f0f6301ae02
-     note:
-   - commit: 7b1d2e3f234bb60948da4144e7bb4f7484e00511
-     note:
+  - commit: 7b1d2e3f234bb60948da4144e7bb4f7484e00511
+    note: Minor MMN bump, adds escaping special characters in mod_log_config
 vccs:
   - commit: 4f1f6d32aa4650700cc7562b150a14eb5ba9631a
-    note:
+    note: Adds cpookie logging in mod_log_config
```

Comment:
Do you mean "cookie" instead of "cpookie"?

Reply:
Yup